2024-02-11 12:38:56,019 DEBUG 4XufSm5mhT : POST /login/customer [in user.py: 64]
2024-02-11 12:38:56,165 DEBUG 4XufSm5mhT : AuthDAO: customer auth and employee auth table created [in authdao.py: 26]
2024-02-11 12:38:56,165 DEBUG 4XufSm5mhT : AuthDAO: tried to fetch user with email diy@gmail.com in table cust_auth [in authdao.py: 34]
2024-02-11 12:38:56,165 INFO 4XufSm5mhT : Customer with c_id:FRnZq logged in [in customer_login_handler.py: 33]
2024-02-11 12:38:56,165 INFO 4XufSm5mhT : Customer FRnZq logged in. [in customer_login_controller.py: 19]
2024-02-11 18:03:09,947 ERROR 9KG2MfjDJK : Employee Login: Invalid email kelly@manager.com provided. [in employee_login_handler.py: 22]
2024-02-11 18:04:56,343 ERROR 5ue3WoEvSC : Error while adding feedback for ticket Np6HU. Error 1136 (21S01): Column count doesn't match value count at row 1 [in feedback_handler.py: 44]
2024-02-11 18:07:05,612 ERROR 5wvgCFkxZp : Error while adding feedback for ticket Np6HU. Error 1136 (21S01): Column count doesn't match value count at row 1 [in feedback_handler.py: 44]
2024-02-11 23:01:38,323 DEBUG G9zWJSApuU : POST /login/employee [in user.py: 77]
2024-02-11 23:01:38,612 DEBUG G9zWJSApuU : AuthDAO: customer auth and employee auth table created [in authdao.py: 26]
2024-02-11 23:01:38,614 DEBUG G9zWJSApuU : AuthDAO: tried to fetch user with email micheal@manager.com in table emp_auth [in authdao.py: 34]
2024-02-11 23:01:38,616 INFO G9zWJSApuU : Employee with e_id:LLYWi, role:manager logged in [in employee_login_handler.py: 31]
2024-02-11 23:01:38,619 INFO G9zWJSApuU : Employee with email micheal@manager.com logged in. [in employee_login_controller.py: 18]
2024-02-11 23:02:18,192 DEBUG CJsysZNTcZ : POST /login/employee [in user.py: 77]
2024-02-11 23:02:18,204 DEBUG CJsysZNTcZ : AuthDAO: customer auth and employee auth table created [in authdao.py: 26]
2024-02-11 23:02:18,205 DEBUG CJsysZNTcZ : AuthDAO: tried to fetch user with email micheal@manager.com in table emp_auth [in authdao.py: 34]
2024-02-11 23:02:18,206 INFO CJsysZNTcZ : Employee with e_id:LLYWi, role:manager logged in [in employee_login_handler.py: 31]
2024-02-11 23:02:18,207 INFO CJsysZNTcZ : Employee with email micheal@manager.com logged in. [in employee_login_controller.py: 18]
2024-02-11 23:04:30,234 DEBUG FiJB5SDHLk : POST /login/employee [in user.py: 77]
2024-02-11 23:04:30,245 DEBUG FiJB5SDHLk : AuthDAO: customer auth and employee auth table created [in authdao.py: 26]
2024-02-11 23:04:30,246 DEBUG FiJB5SDHLk : AuthDAO: tried to fetch user with email micheal@manager.com in table emp_auth [in authdao.py: 34]
2024-02-11 23:04:30,246 INFO FiJB5SDHLk : Employee with e_id:LLYWi, role:manager logged in [in employee_login_handler.py: 31]
2024-02-11 23:04:30,247 INFO FiJB5SDHLk : Employee with email micheal@manager.com logged in. [in employee_login_controller.py: 18]
2024-02-12 10:45:59,693 DEBUG DQZGCt4nBG : POST /login/employee [in user.py: 77]
2024-02-12 10:45:59,822 DEBUG DQZGCt4nBG : AuthDAO: customer auth and employee auth table created [in authdao.py: 26]
2024-02-12 10:45:59,823 DEBUG DQZGCt4nBG : AuthDAO: tried to fetch user with email micheal@manager.com in table emp_auth [in authdao.py: 34]
2024-02-12 10:45:59,824 INFO DQZGCt4nBG : Employee with e_id:LLYWi, role:manager logged in [in employee_login_handler.py: 31]
2024-02-12 10:45:59,825 INFO DQZGCt4nBG : Employee with email micheal@manager.com logged in. [in employee_login_controller.py: 18]
